tests/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixerTest.php

```diff
@@ -642,6 +642,31 @@ public function testFnc()
         {
             aaa();
         }
+    }',
+            ],
+            'expecting exception with message below' => [
+                '<?php
+    class MyTest extends TestCase
+    {
+        /**
+         */
```

Comment:
we should keep the original `Testing stuff.` comment

Reply:
This is because `DocBlock` build annotation in as "everything from `@` to next `@` (or comment end)" - should I change it there or only in this fixer?

Reply:
good point about annotation end.

then, let's use proper docblock in the test:
```
        /**
         * Testing stuff.
         *
         * @expectedException Foo\Bar
         */
```

Reply:
@keradus not really, this PR is about handling that case.